src/python/WMCore/Services/ReqMgrAux/ReqMgrAux.py

```diff
@@ -229,6 +229,72 @@ def deleteConfigDoc(self, docType, docName):
         else:
             return self["requests"].delete('%sconfig/%s' % (docType, docName))[0]['result']
 
+    def getTransferInfo(self):
+        """
+        Get transfer document from coudhdb backend. The document has the following form:
+        {
+          "wf_A": [record1, record2, ...],
+          "wf_B": [....],
+        }
+        where each record has the following format:
+        {"timestamp":000, "dataset":"/a/b/c", "campaignType": "primary", "subscription": subscription}
```

Comment:
replace `campaignType` by `dataType` and `subscription` by `transferIDs`.
As discussed and agreed here: https://gist.github.com/amaltaro/72599f995b37a6e33566f3c749143154#file-transferor_format-json-L24

Reply:
Alan, I already had transferIDs, but then used subscription since I was not sure which PhEDEx/Rucio APIs we're going to use. I suggest before making changes here you review and outline explicitly which APIs which should use, either
```
phedex/datasvc/json/prod/requestlist?dataset=/a/b/c
```
which returns ids or
```
phedex/datasvc/json/prod/subscriptions?dataset=/a/b/c
```
which returns subscription details. I think once we will know which APIs to use we'll adjust ReqMgrAux `getTransferInfo` to use appropriate name for the record.